mobile-app/src/components/Board/Board.tsx

```diff
@@ -7,23 +7,26 @@ import {
   getApproximateSvgSize,
   generateBoardCells,
   CELL_STROKE_COLOR,
-  mapStoneColorToPlayerName,
 } from './boardService';
-import { updateGameState } from '../../services/localplayService';
 import { View, Text } from 'react-native';
 
 interface BoardProps {
   gameState: GameState;
   onCellPress: (coordinates: Coordinates) => void;
+  getPlayersTurnElt: (gameState: GameState) => JSX.Element;
+  getWinMsgElt: (gameState: GameState) => JSX.Element;
 }
 
-export default function Board({ gameState, onCellPress }: BoardProps) {
+export default function Board({
+  gameState,
+  onCellPress,
+  getPlayersTurnElt,
+  getWinMsgElt,
+}: BoardProps) {
   const svgSize = getApproximateSvgSize(gameState.board.length);
   return gameState.winner ? (
     <View style={{ flex: 1, alignItems: 'center', justifyContent: 'center' }}>
-      <Text style={{ fontSize: 32 }}>
-        Player {mapStoneColorToPlayerName(gameState.winner)} has won !
-      </Text>
+      <View style={{ height: '100px' }}>{getWinMsgElt(gameState)}</View>
```

Comment:
I think we can now extract that redundant part and wrap children inside a React Fragment
```tsx
<View style={{ flex: 1, alignItems: 'center', justifyContent: 'center' }}>
```

Reply:
done